src/common/types.ts

```diff
@@ -9,9 +9,33 @@ export type PromotePipelineResult = {
   status: string;
 };
 
+export type Pipeline = {
+  sf_devops__Project__c: string;
+};
+
+export type PipelineStage = {
+  Id: string;
+  sf_devops__Branch__r: Branch;
+  sf_devops__Pipeline__r: Pipeline;
+  sf_devops__Pipeline_Stages__r?: PreviousPipelineStages;
+};
```

Comment:
Future thought.  Can we share these type definitions with the Heroku app?  Might be nice one day.  Maybe a doce-types library?

Reply:
Good idea, I can make do a poc.